src/nlr/LPFormulator.h

```diff
@@ -81,6 +84,49 @@ class LPFormulator : public ParallelSolver
     bool _cutoffInUse;
     double _cutoffValue;
 
+    /*
```

Comment:
Why can't the same thread argument be reused?

Reply:
Reused now.